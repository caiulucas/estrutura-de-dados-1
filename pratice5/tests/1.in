7
Jones
Pedro
Carlos
Lucas
Juca
Valdineia
Jovander
