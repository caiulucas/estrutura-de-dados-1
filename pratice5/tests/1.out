Jovander
Valdineia
Juca
Lucas
Carlos
Pedro
Jones